frontend/cypress/e2e/complaint-search.v2.cy.ts

```diff
@@ -146,3 +147,45 @@ describe("Complaint Search Functionality", () => {
       });
   });
 });
+
+/**
+ * Test that CEEB specific search filters work
+ */
+describe("Verify CEEB specific search filters work", () => {
+  beforeEach(function () {
+    cy.viewport("macbook-16");
+    cy.kcLogout().kcLogin(Roles.CEEB);
+  });
+
+  it("allows filtering of complaints by Action Taken", function () {
+    // Navigate to the complaint list
+    const complaintWithActionTakenID = "23-030990";
+    const actionTaken = "Forward to lead agency";
+    cy.visit("/");
+    cy.waitForSpinner();
+
+    // Set an 'action taken' on a complaint, so it can be filtered
+    cy.get("#comp-officer-filter").should("exist").click();
+    cy.get(`#${complaintWithActionTakenID}`).should("exist").click();
```

Comment:
We have a shortcut function that will allow you navigate directly to a complaint: cy.navigateToDetailsScreen might speed up the test a bit.